tests/test.py

```diff
@@ -396,6 +396,22 @@ def get_poison_record():
     return poison_record
 
 
+def get_new_poison_record(extra_kwargs: dict=None, keys_dir=None):
+    """generate one new poison record for speed up tests and don't create subprocess
+    for new records"""
+    args = [os.path.join(BINARY_OUTPUT_FOLDER, 'acra-poisonrecordmaker')]
+    if keys_dir:
+        args.append('--keys_dir={}'.format(keys_dir))
```

Comment:
should we pass here keystore version too?

Reply:
No, its is required for `keymaker` but not for `poisonrecordmaker`.